js/ui/handler/scroll_zoom.js

```diff
@@ -37,14 +37,23 @@ class ScrollZoomHandler {
     /**
      * Enables the "scroll to zoom" interaction.
      *
+     * @param {boolean|Object} options If {around: "center"} is passed, map will zoom around center of map
```

Comment:
`boolean|Object` here is incorrect. `options` should be documented as an optional object, and `options.around` as a separate subitem.